package.json

```diff
@@ -79,8 +79,7 @@
   },
   "dependencies": {
     "asap": "2.0.3",
-    "falcor-json-graph": "1.1.5",
-    "falcor-path-syntax": "0.2.1",
+    "falcor-json-graph": "2.0.0",
```

Comment:
Just updating at the same time? I assume this is unrelated to the path-syntax removal. It's fine if we are, just wanted to make sure it's conscious.